pkg/inspector/image-inspector.go

```diff
@@ -124,6 +125,8 @@ func (i *defaultImageInspector) Inspect() error {
 		return fmt.Errorf("Unable to connect to docker daemon: %v\n", err)
 	}
 
+	ctx := context.Background()
```

Comment:
do we want to make use of a useful context here? e.g. `context.WithTimeout`? It might be useful to provide a CLI flag that allows configuring a scan timeout, which we can then pass to `context.WithTimeout`

Reply:
@ilackarms I'm not sure that the timeout is acceptable here because the speed of `Inspect` depends on the download time of the image (network speed), on the disk speed.

The timeout could be configurable. I think that such things are worth doing as follow up.